steam_utils.py

```diff
@@ -28,23 +30,25 @@ def __init__(self, path: str):
         self.games = []
         for filename in os.listdir(os.path.join(path, "steamapps")):
             if filename.startswith("appmanifest"):
-                with open(
-                    os.path.join(path, "steamapps", filename), "r", encoding="utf-8"
-                ) as fp:
-                    i, n = None, None
-                    for line in fp:
-                        line = line.strip()
-
-                        if line.startswith('"appid"'):
-                            i = line.replace('"appid"', "").strip()[1:-1]
-                        if line.startswith('"installdir"'):
-                            n = line.replace('"installdir"', "").strip()[1:-1]
-
-                        if i is not None and n is not None:
-                            break
-                if i is None or n is None:
-                    continue
-                self.games.append(SteamGame(i, n))
+                filepath = os.path.join(path, "steamapps", filename)
+                try:
+                    with open(filepath, "r", encoding="utf-8" ) as fp:
+                        i, n = None, None
+                        for line in fp:
+                            line = line.strip()
+
+                            if line.startswith('"appid"'):
+                                i = line.replace('"appid"', "").strip()[1:-1]
+                            if line.startswith('"installdir"'):
+                                n = line.replace('"installdir"', "").strip()[1:-1]
+
+                            if i is not None and n is not None:
+                                break
+                    if i is None or n is None:
+                        continue
+                    self.games.append(SteamGame(i, n))
+                except:
```

Comment:
Minor things:
- The linter will yell for a bare `except`, I'd go for `except UnicodeDecodeError:`.
- You can use `print(..., file=sys.stderr)` to avoid importing Qt stuff.